### ISSUE TITLE
Feature request: On the confirm buy now dialogue script, bind a keyboard hotkey to 'buy now' on current selected item.

### ISSUE THREAD

**Issue by ThinWhiteDuke87** [Mardaneus86/futwebapp-tampermonkey#37]
Would help a lot to have it on a hot key.

**Comment by Mardaneus86** [comment]
We could distribute [@martellaj's script](https://github.com/martellaj/shortfuts/blob/d74c15e8734ebc5aa548d8a699c4bf7d25fbb4e6/script.js) through a Tampermonkey script if that's helpful. That would make it available in all browsers.

It contains an MIT license, so it's free to distribute as long as the credit goes to him. Perhaps we could just load the script directly from Tampermonkey by referencing to the [github repo](https://github.com/martellaj/shortfuts).

**Comment by ThinWhiteDuke87** [comment]
Yes that would be helpful. However i've looked at the script, and there is no "press buy now" hot key in the current script. It would need an added hotkey?
